client/src/components/NetworkDashboard.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+const NetworkDashboard = () => {
+    //Obtain Available Network Information from User's Browser
+    const networkInterface = navigator.connection 
+                             || navigator.mozConnection 
+                             || navigator.webkitConnection;
+
+    //Creates an Array of Arrays of the Network Data for Mapping
+    let network = [];
+
+    if(networkInterface !== undefined) {
```

Comment:
I really like what you're doing here but could you add some info on what the new keys 'mean' to the laymen? What may eventually turn into a tooltip?